can/interfaces/__init__.py

```diff
@@ -3,42 +3,57 @@
 """
 
 import sys
-from typing import Dict, Tuple
+from typing import cast, Dict, Iterable, Tuple
 
 # interface_name => (module, classname)
-BACKENDS: Dict[str, Tuple[str, ...]] = {
-    "kvaser": ("can.interfaces.kvaser", "KvaserBus"),
-    "socketcan": ("can.interfaces.socketcan", "SocketcanBus"),
-    "serial": ("can.interfaces.serial.serial_can", "SerialBus"),
-    "pcan": ("can.interfaces.pcan", "PcanBus"),
-    "usb2can": ("can.interfaces.usb2can", "Usb2canBus"),
-    "ixxat": ("can.interfaces.ixxat", "IXXATBus"),
-    "nican": ("can.interfaces.nican", "NicanBus"),
-    "iscan": ("can.interfaces.iscan", "IscanBus"),
-    "virtual": ("can.interfaces.virtual", "VirtualBus"),
-    "udp_multicast": ("can.interfaces.udp_multicast", "UdpMulticastBus"),
-    "neovi": ("can.interfaces.ics_neovi", "NeoViBus"),
-    "vector": ("can.interfaces.vector", "VectorBus"),
-    "slcan": ("can.interfaces.slcan", "slcanBus"),
-    "robotell": ("can.interfaces.robotell", "robotellBus"),
-    "canalystii": ("can.interfaces.canalystii", "CANalystIIBus"),
-    "systec": ("can.interfaces.systec", "UcanBus"),
-    "seeedstudio": ("can.interfaces.seeedstudio", "SeeedBus"),
-    "cantact": ("can.interfaces.cantact", "CantactBus"),
-    "gs_usb": ("can.interfaces.gs_usb", "GsUsbBus"),
-    "nixnet": ("can.interfaces.nixnet", "NiXNETcanBus"),
-    "neousys": ("can.interfaces.neousys", "NeousysBus"),
-    "etas": ("can.interfaces.etas", "EtasBus"),
-    "socketcand": ("can.interfaces.socketcand", "SocketCanDaemonBus"),
-}
+BACKENDS: Dict[str, Tuple[str, str]] = cast(
```

Comment:
Locally, it works without the cast. Are you sure it's necessary?

Reply:
I'm setting it up locally and push something proper ^^ Maybe I messed something up between the changes.

Reply:
Yeah, works locally without 👍